RELEASE/scripts/presool.ash

```diff
@@ -59,6 +59,27 @@ void handlePreAdventure(location place)
 		}
 	}
 
+	if(sl_have_familiar($familiar[cat burglar]) && sl_is_valid($familiar[cat burglar]))
```

Comment:
`sl_have_familiar` includes an `sl_is_valid` check already.

Reply:
ugh this again sorry yes